docs/dpg-archboard.md

```diff
@@ -0,0 +1,350 @@
+   be able to accessed as dicts and as models as well
+3. We've added glass breaker `send_request` to all of our clients. Clients can now send an `HttpRequest` directly to the server leveraging our client pipelines.
+4. We've also added easy-to-use customizations, and are going to be pushing these customizations for SDK authors.
+
+
+--------------------------------------------------------------------------
+
+--------------------------------- Yuchao ---------------------------------
+
+## Developer Experience
+
+### Creating A Client
+
+yuchao-note: endpoint order
+
+Creating A Client is largely the same as it was before. We have made the client arguments align with the Python guidelines, generating an `endpoint` and `credential` parameter.
+
+```python
+from azure.purview.catalog import PurviewCatalogClient
+from azure.identity import DefaultAzureCredential
+
+client = PurviewCatalogClient("http://myendpoint.com", DefaultAzureCredential())
+```
+
+#### Creating A Client - API Diff
+
+![Screen Shot 2022-05-11 at 11 24 07 AM](https://user-images.githubusercontent.com/43154838/167919888-81d5cf38-1e8e-4a50-9d1c-5f01df972a6b.png)
+
+#### Creating A Client - User Behavior Diff
+
+- Legacy
+
+```python
+from azure.purview.catalog import PurviewCatalogClient
+from azure.identity import DefaultAzureCredential
+    
+client = PurviewCatalogClient(DefaultAzureCredential(), "http://my-endpoint.com")
+```
+
+- DPG
+
+```python
+from azure.purview.catalog import PurviewCatalogClient
+from azure.identity import DefaultAzureCredential
+    
+client = PurviewCatalogClient("http://my-endpoint.com", DefaultAzureCredential())
+```
+
+
+
+### Simple GET/DELETE Request
+
+yuchao-note: 1. return type 2. Api change: keyword-only
+
+GET/DELETE requests are largely the same as before. Since a lot of Azure services are JSON based and return JSON payloads, here is a main place where we will see differences between GET/DELETE requests with current generated code, because we no longer return models in this case.
+We return raw JSON, so in most cases this means users access them like a dict. Python users have been very receptive to just accessing these
+models as JSON, since Python users are already very comfortable with JSON. In fact, we've been getting issues over the years asking for our models
+to really be dicts, so this change has actually been quite welcomed.
+
+We still see the importance of adding DPG models, in large part because of the documentation benefits they give us, they serve
+as a landing place for documentation. But when we add them, they will be non-breaking because you can still access them as a dict.
+
+```python
+...
+response = client.delete_by_unique_attribute("type_name", attr_qualified_name="attr_qualified_name")
+print(response["guidAssignments"])
+```
+
+#### Simple GET/DELETE Request - API Diff
+
+![Screen Shot 2022-05-11 at 11 28 26 AM](https://user-images.githubusercontent.com/43154838/167920536-02f11833-23c9-4446-b8fd-452529223a34.png)
```

Comment:
Hi @iscai-msft , Wondering how is the red deleted part generated? I see 'cls' as positional parameter in the pictures, but it's in kwargs in our current legacy generation.